mmseg/datasets/dataset_wrappers.py

```diff
@@ -188,3 +190,85 @@ def __getitem__(self, idx):
     def __len__(self):
         """The length is multiplied by ``times``"""
         return self.times * self._ori_len
+
+
+@DATASETS.register_module()
+class MultiImageMixDataset:
+    """A wrapper of multiple images mixed dataset.
+
+    Suitable for training on multiple images mixed data augmentation like
+    mosaic and mixup. For the augmentation pipeline of mixed image data,
+    the `get_indexes` method needs to be provided to obtain the image
+    indexes, and you can set `skip_flags` to change the pipeline running
+    process.
+    Args:
```

Comment:
```suggestion

    Args:
```

Reply:
`pre-commit run --all-files`

```
flake8 .... Failed

line 204: W293 blank line contains whitespace
```

Is this okay?

Reply:
Just add a blank line, there might be some whitespace.
Please refer
https://github.com/open-mmlab/mmsegmentation/blob/ff9d695b4051e9f2384da11ba4dbdef5d003a856/mmseg/datasets/dataset_wrappers.py#L41-L54